examples/set-label/functions/fn-config.yaml

```diff
@@ -8,5 +8,12 @@ metadata:
         image: gcr.io/kpt-functions/set-label
     config.kubernetes.io/local-config: 'true'
 data:
-  label_name: color
-  label_value: orange
+  specs:
+  - label_name: color
+    label_value: orange
+  - label_name: fruit
+    label_value: apple
+    fieldSpecs:
```

Comment:
It seems this is per-label config. Why? Should it apply to all of the labels?

Reply:
The default config is applied to all labels. Each label can have it's own custom spec.

Reply:
I see. Thanks for clarification.